ja_jp.lang

```diff
@@ -239,23 +283,252 @@ gui.xaero_sort_unsorted=なし
 gui.xaero_sort_name=名前
 gui.xaero_sort_symbol=頭文字
 gui.xaero_sort_distance=距離
+gui.xaero_box_cave_maps=地下にある洞窟や建物の構造を表示するために、表面よりも深いブロックを操作するマッピングモードです。 このサイズを超える固体ブロックの正方形をプレイヤーの頭上で検知したときに有効化されます。
+gui.xaero_box_cave_maps2=地下にある洞窟や建物の構造を表示するために、表面よりも深いブロックを操作するマッピングモードです。 このサイズを超える固体ブロックの正方形をプレイヤーの頭上で検知したときに有効化されます。 この固体ブロックは同じYにある必要はありません。
 gui.xaero_box_entity_amount=エンティティの種類ごとの上限
+gui.xaero_box_distance=ゲーム内で距離を表示するウェイポイントを指定します。
+gui.xaero_box_distance2=ゲーム内で距離を表示するウェイポイントを指定します。 一度の複数のウェイポイントに表示されることを設定するものではありません。 そのような用途には\"複数ウェイポイントの距離表示\"を利用してください。
```

Comment:
Weird wording when translated back but could be correct. Please explain what these lines mean to you.